### ISSUE TITLE
Staging Review [Public Websites - Harassment Reporting Tool (HRT) Content Landing Page]

### ISSUE THREAD

**Issue by brianalloyd** [department-of-veterans-affairs/va.gov-team#29178]
## Steps to complete Staging Review: 

- [x] VFS Product Manager: Create this issue and fill in your team and feature name in the title.
- [x] VFS: Using the Slack Workflow ([User guide](https://depo-platform-documentation.scrollhelp.site/support/Getting-help-from-the-Platform-in-Slack.1439138197.html)), notify Collaboration Cycle team of this Github ticket in #vfs-platform-support. 
- [x] Platform: Schedule meeting with Platform reviewers and requesting team attendees (as listed below)
- [x] VFS: At least 4 days before scheduled meeting, you must complete list of artifacts below
- [x] VFS: Verify that access information has been provided for the tool or feature in staging. Do not put staging credentials in your va.gov-team ticket; store or reference them in a .md file in the va.gov-team-sensitive repository.
- [ ] Platform: Create feedback tickets and link to the Staging Review ticket (if no feedback, please comment stating that). 
- [ ] VFS: Respond to Platform feedback tickets.
- [ ] VFS: Update this ticket with the product’s launch date after it has gone live.
- [ ] Platform: Update [Platform Collaboration Point Tracker](https://docs.google.com/spreadsheets/d/1OgPyEvUlNF6EnaYMFAXJkV6FKOvZnlPnbOQ2fAJ7W7A/edit#gid=266151061).
- [ ] VFS: Close this ticket when the review is complete.

## Artifacts
- Link to [product outline](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/harassment-reporting/product-outline.md)
- Link to the tool or feature in [Drupal](http://preview-prod.vfs.va.gov/preview?nodeId=34641)
[How To Report Harassment At VA _ Veterans Affairs.pdf](https://github.com/department-of-veterans-affairs/va.gov-team/files/7196438/How.To.Report.Harassment.At.VA._.Veterans.Affairs.pdf)

-  Link to PDF Version of the tool.
[How To Report Harassment At VA _ Veterans Affairs.pdf](https://github.com/department-of-veterans-affairs/va.gov-team/files/7196436/How.To.Report.Harassment.At.VA._.Veterans.Affairs.pdf)
[content](http://preview-prod.vfs.va.gov/preview?nodeId=34641)
-  Link to or attach your project's content source of truth (https://sketch.com/s/b3f90417-a2a0-41f5-8fb9-151ce163710a) 
-  Link to the Github ticket that shows the work for creating and/or updating any relevant static content pages and entry points for this project
- Link to [release plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/harassment-reporting/HRT%20Release%20Guide.md) with the "Planning" sections completed (in each section: Phase I, Phase II, Go Live)
- Link to your product test [case](https://dsvavsp.testrail.io/index.php?/cases/view/6181&group_by=cases:section_id&group_order=asc&display_deleted_cases=0&group_id=1518)/test [plan](https://dsvavsp.testrail.io/index.php?/plans/view/2153) in TestRail with test execution logs ([learn more](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/quality-assurance/qa-artifacts.md#test-plan)) _for QA practice area review_
- \[No reports for this content-page.  There're neither any varying use-cases nor any defects logged from test-runs.\]
- Link to verification that accessibility checks are complete:
  - (preferred) Link to your [TestRail Platform accessibility test plan template](https://dsvavsp.testrail.io/index.php?/suites/view/14&group_by=cases:section_id&group_order=asc).
OR
  - Link to a doc in your product folder that indicates all required checks from the accessibility [staging review prep](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/staging-review-processes.md) doc are complete.
- List of known issues (with links to tickets when applicable)

## Meeting attendees from requesting team
- Product Manager (required): Brian Lloyd 
- OCTO-DE Product lead (recommended): David Conlon
- OCTO-DE Design lead (required): Ryan Thurlwell
- Anyone else from your team who significantly contributed to the artifacts provided (required): name(s) Cassandra Allen, Kelson Adams, Ryan Leahy
- VFS Content, IA, accessibility, and QA specialists (if applicable): name(s) Danielle Thierry, Martha Wilkes Soni Mechem (PIF/Stakeholder) 

### Staging Reviews can only be scheduled on Thursday.

Please select your preferred time slot per the [Platform Collaboration Meetings Calendar](https://calendar.google.com/calendar/u/0/embed?src=adhocteam.us_4dn3o77gcm5e3vbiedlha96tc0@group.calendar.google.com&ctz=America/New_York).<br>

**09/09 2PM **
 
Please note:
- All times are shown in ET. 
- Available time slots are indicated with [STAGING BLOCK]. 
- When you reserve a time slot, the reserved meeting will show up alongside the [STAGING BLOCK]. 
- Please select a time slot at least 4 business days from request date.

## Additional information
Please refer to [Platform Collaboration Cycle](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/index.html) on Platform website for more information about the Collaboration Cycle.

**Comment by shiragoodman** [comment]
Staging Review has been scheduled for 9/9 at 2:00pm ET.

**Comment by tlei123** [comment]
[Test runs](https://dsvavsp.testrail.io/index.php?/plans/view/2153) nominally scheduled for Thu 9/2, pending status-check w/ team tomorrow.

**Comment by tlei123** [comment]
> @tlei123 I added the link to content source of truth: http://preview-prod.vfs.va.gov/preview?nodeId=34641 let me know if any questions remain. Thanks.

@brianalloyd , I just noticed what @DanielleThierryUSDSVA [gave me as the page-under-test](https://dsva.slack.com/archives/C52CL1PKQ/p1630429263129500?thread_ts=1630005326.020400&cid=C52CL1PKQ) is the same URL as what you provided above as copy-specs -- they cannot be the same.

Do we have a separate copy requirements-doc?

**Comment by tlei123** [comment]
Per our [Slack convo](https://dsva.slack.com/archives/C52CL1PKQ/p1630609166193800?thread_ts=1630005326.020400&cid=C52CL1PKQ), page-under-test & copy source of truth are actually the same URL.  Will do the usual informal proof-read then.

**Comment by shiragoodman** [comment]
Staging Review moved to 9/16 per PM request.

**Comment by brianalloyd** [comment]
@shiragoodman @andreahewitt-odd in recent communication with stakeholder we're looking to shift the staging review to 9/23 if possible based on some needed content from VA Stakeholder in order to finalize the static content landing page.  I posted in slack but also wanted to follow up on this ticket in support.

**Comment by allison0034** [comment]
@brianalloyd 9/23 is available. Shira is back in the office today and owns the meeting, she will get it updated. Thanks you!

**Comment by shiragoodman** [comment]
@brianalloyd Staging REview has been moved to 9/23 at 2:00pm. 

Thanks @allison0034 !

**Comment by allison0034** [comment]
@brianalloyd I do not see a file online option in the link or PDF. I see it in Sketch. Will we be reviewing an online way to report for this? Or are we just reviewing the page the pdf page?

**Comment by AngelaFowler82** [comment]
Hello all, I do not see this component in staging. As the review is Thursday and we have only until Wednesday morning to provide feedback, Is there a plan to move it to staging by tomorrow?

**Comment by DanielleThierryUSDSVA** [comment]
Hi all. My understanding that this review is only for the [landing page](http://preview-prod.vfs.va.gov/preview?nodeId=34641) at this time. The online tool comes next in a separate iteration. cc @brianalloyd

**Comment by AngelaFowler82** [comment]
I understand, but will the landing page be in staging in preparation for the Staging Review?

**Comment by DanielleThierryUSDSVA** [comment]
Ah, good question, @AngelaFowler82. I saw Allison's question above about the online tool and thought you were also referring to the tool. The answer to the staging question is no. This landing page is a new Drupal page, and Drupal pages go straight to production. So we review Drupal pages in preview instead of staging. 

Also, to note: Links to other pages often do not work in the preview environment so they will appear broken. The sitewide content team validates links within Drupal and then again on the live site after publishing.

**Comment by brianalloyd** [comment]
> Hi all. My understanding that this review is only for the [landing page](http://preview-prod.vfs.va.gov/preview?nodeId=34641) at this time. The online tool comes next in a separate iteration. cc @brianalloyd

That is exactly correct, review is only for the landing page at this time.

**Comment by tlei123** [comment]
@pjhill @ddzz , QA test-runs now completed.  Test plan & case links updated above in main description.

100% Passed -- no defects logged. 

No reports generated, since there're no defects logged, and no varying use-cases for content.

cc @brianalloyd @shiragoodman

**Comment by pjhill** [comment]
@tlei123 Thanks for producing those test plan and execution record artifacts. Darius and I went through the product and test plan, and we have no feedback. Thanks all!

**Comment by DanielleThierryUSDSVA** [comment]
Hi @shiragoodman @wesrowe @jilladams. I'm resurrecting this ticket because the phone system issue that had caused us to have to pause this work a long while back has been resolved and we are now cleared by our VA partners to launch this page. (Wes and Jill, for background, this started as a PW + Content/IA team project because it started as a full interactive tool experience. We then had to pull it back to be just a content landing page to start. This launch is **only** for the content landing page, not the full tool. After we publish the page, we'll need to connect on adding the link to the footer).

I'll go through the old staging review tickets here and reach out with questions, etc. so we can get this important page launched for Veterans.

cc @davidconlon @artsymartha68

**Comment by DanielleThierryUSDSVA** [comment]
Hi @shiragoodman @wesrowe @jilladams. I'm resurrecting this ticket because the phone system issue that had caused us to have to pause this work a long while back has been resolved and we are now cleared by our VA partners to launch this page. (Wes and Jill, for background, this started as a PW + Content/IA team project because it started as a full interactive tool experience. We then had to pull it back to be just a content landing page to start. This launch is **only** for the content landing page, not the full tool. After we publish the page, we'll need to connect on adding the link to the footer).

I'll go through the old staging review tickets here and reach out with questions, etc. so we can get this important page launched for Veterans.

cc @davidconlon @artsymartha68
